docs/requirements.yaml

```
@@ -11,5 +11,5 @@ dependencies:
   - ipython
   - MDAnalysisTests
     # Pip-only installs
-  #- pip:
-
+  - pip:
+  - nbsphinx
```

Shouldn't this bee
```suggestion
     - nbsphinx 
```
i.e., indented so that it forms a list as the dict item `{'pip': ['nbsphinx']}` ?

Don't just take this suggestion but have a look at what the exact format of the requirements file ought to be.